phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -1728,6 +1739,11 @@ public MutationState createCDC(CreateCDCStatement statement) throws SQLException
                         SortOrder.getDefault()) }));
         IndexType indexType = (IndexType) TableProperty.INDEX_TYPE.getValue(tableProps);
         ListMultimap<String, Pair<String, Object>> indexProps = ArrayListMultimap.create();
+        if (TableProperty.SALT_BUCKETS.getValue(tableProps) != null) {
+            indexProps.put(QueryConstants.ALL_FAMILY_PROPERTIES_KEY, new Pair<>(
+                    TableProperty.SALT_BUCKETS.getPropertyName(),
+                    TableProperty.SALT_BUCKETS.getValue(tableProps)));
+        }
```

Comment:
do we also do this for existing indexes?

Reply:
i just revisited the design doc to confirm that we need this option for uncovered index